errorHandler.js

```diff
@@ -0,0 +1,53 @@
+const { HttpCodes } = require("./messages/httpCodes");
+const { customErrors } = require("./messages/errors");
+const { messages } = require("./messages/messages");
+
+class MessageToUser {
+    constructor(name) {
+        this.statusCode = messages[name].code;
```

Comment:
It seems to me very complicated. Lets make it simpler:
hadrcode here 200 success full message and expect on arguments just messageString.
So the usage will be like `new MessageToUser(USER_CREATED_MESSAGE)`.